src/main/scala/nl.knaw.dans.easy.license/Command.scala

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.license
+
+import java.io.FileOutputStream
+
+import nl.knaw.dans.lib.error._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import resource._
+
+import scala.language.reflectiveCalls
+import scala.util.control.NonFatal
+import scala.util.{ Failure, Success, Try }
+import internal._
+
+object Command extends App with DebugEnhancedLogging {
+  type FeedBackMessage = String
+
+  val configuration = Configuration()
+  val commandLine: CommandLineOptions = new CommandLineOptions(args, configuration) {
+    verify()
+  }
+  val app = new LicenseCreatorApp(configuration)
+  managed(app)
+    .acquireAndGet(app => {
+      for {
+        msg <- runSubcommand(app)
+      } yield msg
+    })
+//    .doIfSuccess(msg => println(s"OK: $msg"))
```

Comment:
not sure why these wouldn't compile. Did you use dans-scala-lib version 1.2? Did you import the correct package? What is the type of the expression before it? What is the compile error message you get?

Reply:
Yes, I am using version 1.2. IntelliJ even underlines the implicit methods and jumps to the definition with Cmd-B.

Reply:
The type of `managed(app).acquireAndGet(runSubcommand)` is `Try[Command.FeedbackMessage]`.

Reply:
Error message:
```
[INFO] Compiling 21 source files to /Users/janm/git/service/easy/easy-license-creator/target/classes at 1504101538746
[ERROR] /Users/janm/git/service/easy/easy-license-creator/src/main/scala/nl.knaw.dans.easy.license/Command.scala:40: error: value doIfSuccess is not a member of scala.util.Try[nl.knaw.dans.easy.license.Command.FeedBackMessage]
[ERROR] possible cause: maybe a semicolon is missing before `value doIfSuccess'?
[ERROR]     .doIfSuccess(msg => println(s"OK: $msg"))
[ERROR]      ^
```

Reply:
Imports:
```
package nl.knaw.dans.easy.license

import java.io.FileOutputStream

import nl.knaw.dans.lib.error._
import nl.knaw.dans.lib.logging.DebugEnhancedLogging
import resource._

import scala.language.reflectiveCalls
import scala.util.control.NonFatal
import scala.util.{ Failure, Success, Try }
import internal._
```

Reply:
Ah, wait a minute ... there is also a `TryExtensions` in the `internal` package ...

Reply:
Right, that was it. Scala apparently doesn't know how to mix all these exentions methods together.